pkg/api/v1alpha1/cluster_types.go

```diff
@@ -420,6 +422,21 @@ func (n *Endpoint) CloudStackEqual(o *Endpoint) bool {
 	return n.Host == o.Host
 }
 
+type ApiServerExtraArgs struct {
```

Comment:
I thought we were doing a map and not a struct right?

Reply:
Yes, forgot to update the branch